src/main/java/com/artipie/SliceFromConfig.java

```diff
@@ -250,15 +250,12 @@ static Slice build(
                 );
                 break;
             case "npm-proxy":
-                slice = trimIfNotStandalone(
-                    settings, standalone,
-                    new NpmProxySlice(
-                        cfg.path(),
-                        new NpmProxy(
-                            new NpmProxyConfig(cfg.settings().orElseThrow()),
-                            Vertx.vertx(),
-                            cfg.storage()
-                        )
+                slice = new NpmProxySlice(
+                    cfg.path(),
+                    new NpmProxy(
+                        new NpmProxyConfig(cfg.settings().orElseThrow()),
+                        Vertx.vertx(),
```

Comment:
@genryxy it is not something you added, but there is a big problem in this code. `Vertx.vertx()` call creates a Vert.x instance, a very heavy object. It should be created only once for whole application on start an reused everywhere. Could you please add a puzzle for solving this issue?